Add ability for PC plots wireframe and points mode to be colored by var. (#20595)
visit-dav/visit: src/doc/using_visit/Plots/PlotTypes/PseudocolorPlot.rst

Representation
''''''''''''''

> By default, the **Pseudocolor** plot renders as a **Surfaces**.

Reviewer: ```suggestion
By default, the **Pseudocolor** plot renders **Surfaces**.
```

> It can also render in **Wireframe** or **Points** mode.
> Choose the representation by checking one or any combination of the three.

Reviewer: ```suggestion
It can also render a **Wireframe** or collection of **Points**.
Choose the representation by checking one or more of the three.
```
